CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - /api/1.3/system/info
   - /api/1.3/types (R)
 - Fair Queuing Pacing: Using the FQ Pacing Rate parameter in Delivery Services allows operators to limit the rate of individual sessions to the edge cache. This feature requires a Trafficserver RPM containing the fq_pacing experimental plugin AND setting 'fq' as the default Linux qdisc in sysctl. 
+- Backup Edge Cache group: Backup Edge group for a particular cache group can be configured using Traffic Router's cr-config.json configuration file. With this, users will be able control traffic within portions of their network, thereby avoiding choosing fallback cache groups using geo coordinates(getClosestAvailableCachegroup). This can be controlled using "backupLocations" which contains configuration for backup cache groups parsed from the cr-config.json which gets populated using the APIs detailed [here](http://traffic-control-cdn.readthedocs.io/en/latest/development/traffic_ops_api/v12/cachegroup_fallbacks.html)
```

Comment:
Since the changelog is user visible, its better to describe the changes as configured using Traffic Ops API. You could also make the description more concise as you have more detailed documentation elsewhere. 

Consider something along the lines of:
```
  Configure a backup list of edge cache groups through the Traffic Ops API. A Backup edge cache group is used if the matched group in the CZF is not available. If the backup edge cache groups are not available, geo location can optionally be used as a further backup. See APIs here...
```

Reply:
Will do.